### PR TITLE
Add ProfileAndSettings with route

### LINKED ISSUE #125: Add a profile & settings page placeholder
For now, profile & settings page will be an empty placeholder, but let's already implement the path to get there. 

For this task
- Create a simple placeholder component called `ProfileAndSettings` [here](https://github.com/HandsOnLearning/the-project/tree/master/src/components/Content/components). 
- It should not do much, just something like:
```TypeScript
const ProfileAndSettings = () => <div>Profile & Settings</div>
```
- It should be shown on the content side (right) 
- the route should be `/settings`
- when we expand UserCard, click `Profile & Settings`, it should route us there

You will need to use [`react-router`](https://reacttraining.com/react-router/web/guides/quick-start) for this, and more specifically
- `Link`
- `Switch`
- `Route`

### PR DESCRIPTION
**Pull request by SimonNord**
closes #125 
- Create a new component for Profile and Settings
- Added Link with props to "/settings"
- Centered the text

### CODE REVIEW

**Comment by SimonNord** [comment]
"you should not use Link outside of Router" is the error, but the Router is wrapping the whole app right? Should this solution not work? @MoSattler

**Comment by SimonNord** [comment]
"you should not use Link outside of Router" is the error, but the Router is wrapping the whole app right? Should this solution not work? @MoSattler

File: src/components/Sidebar/components/UserCard/styles.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export const ExpandedList = styled.div`
   left: 0;
 `;
 
-export const ExpandedListItem = styled.a`
+export const ExpandedListItem = styled.div`
```
Comment (MoSattler) [inline src/components/Sidebar/components/UserCard/styles.js:64]: you might run into merge conflicts with @limbovski

**Comment by SimonNord** [comment]
Added <BrowserRouter/> to the test, and they all pass, is that okey?
@MoSattler

File: src/components/Sidebar/components/UserCard/styles.js (hunk 1 above)
Comment (SimonNord) [reply src/components/Sidebar/components/UserCard/styles.js:64]: yeah I think I will, but got console error in chrome about that a tag :P

**Pull request merged**